guides/common/modules/proc_content-view-version-comparison.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Content_View_Version_Comparison_{context}"]
+= Content View Version Comparison
```

Comment:
```suggestion
[id="Comparing_Content_View_Versions_{context}"]
= Comparing Content View Versions
```
Editing title to gerund form by the IBM style guide. Please also update file name